server/controllers/review.js

```diff
@@ -0,0 +1,49 @@
+const Review = require("../models/Review");
+
+// @route GET /reviews/all/:sitterId
+// @desc gets all reviews for sitter
+// @access Private
+exports.getAllReviews = async (req, res) => {
+  const reviews = await Review.find({ sitterId: req.params.sitterId });
```

Comment:
Additionally, you can also add a validation check to see if the request params exist and if they don't return a `400`

Reply:
Since the route is "/reviews/all/:sitterId," wouldn't the route not be found anyways if the sitterId was not provided in the URL? 

Instead, I added in a check to see that the provided sitterId was a valid user.

Reply:
@jb162 Good point, when we validate the route parameters we also want to check whether the param is a valid object id as below (otherwise, we will end up querying the DB or perform other operations only to return an error):

```
  if (!mongoose.Types.ObjectId.isValid(sitterId)) {
    return res.status(400).send("Bad Request");
  }
```